e2e/tests/nightly/genomic-extraction-to-vcf.spec.ts

```diff
@@ -217,16 +217,16 @@ describe('Genomics Extraction Test', () => {
     const genomicSidebar = new GenomicExtractionsSidebar(page);
     const startTime = Date.now();
     while (Date.now() - startTime <= maxTime) {
+      await page.waitForTimeout(1 * 60 * 1000); // sleep 1 min
```

Comment:
Pause 1 minute instead 10 minutes between retries. Because the extraction job could have finished during the pause while test is still waiting.

Reply:
Where did you get 10 minutes? pollInterval is 10 seconds

ALso, I prefer where this sleep was before - the function should immediately check rather than sleeping first.

Reply:
You're right. Reverted this change. Thanks.